src/test/java/nextstep/subway/line/SectionNavigatorTest.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.subway.line;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Section;
+import nextstep.subway.domain.SectionNavigator;
+import nextstep.subway.domain.Station;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SectionNavigatorTest {
+    @DisplayName("구간들을 순서대로 탐색하여 역을 조회")
+    @Test
+    void orderedStations() {
+        Line line = new Line("2호선", "green");
+        Station station1 = new Station("서초역");
+        Station station2 = new Station("논현역");
+        Station station3 = new Station("신도림역");
+        Station station4 = new Station("신림역");
+        Station station5 = new Station("교대역");
+        Station station6 = new Station("강남역");
```

Comment:
해당 테스트 코드내에서만 사용되는 테스트 픽스쳐이기에 좀 더 직관적으로 알아볼 수 있게 명명해도 좋을 것 같네요 :)
```suggestion
        Line line = new Line("2호선", "green");
        Station 서초역 = new Station("서초역");
        Station 논현역 = new Station("논현역");
        Station 신도림역 = new Station("신도림역");
        Station 신림역 = new Station("신림역");
        Station 교대역 = new Station("교대역");
        Station 강남역 = new Station("강남역");
```

Reply:
네 좋습니다!👍🏻